bears/general/FilenameBear.py

```diff
@@ -71,15 +81,32 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if new_name != filename_without_extension:
+            messages.append(
+                'Filename does not follow {} naming-convention.'.format(
+                    file_naming_convention))
+
+        if not filename_without_extension.startswith(filename_prefix):
+            new_name = filename_prefix + new_name
+            messages.append(
+                'Filename does not use the {!r} prefix.'.format(
+                    filename_prefix))
+
+        if not filename_without_extension.endswith(filename_suffix):
+            new_name = new_name + filename_suffix
+            messages.append(
+                'Filename does not use the {!r} suffix.'.format(
+                        filename_suffix))
```

Comment:
I was embarrassed that i still have indentation problems but i think this message is the reason why i indented this line.
```
bears/general/FilenameBear.py                                                                                                         
|  99| ················filename_suffix))
|    | [NORMAL] PycodestyleBear (E122):
|    | E122 continuation line missing indentation or outdented
|    | *0: Do nothing
|    |  1: Open file(s)
|    |  2: Add ignore comment

```